pkg/estimator/pb/types.go

```diff
@@ -47,3 +49,45 @@ type MaxAvailableReplicasResponse struct {
 	// +required
 	MaxReplicas int32 `json:"maxReplicas" protobuf:"varint,1,opt,name=maxReplicas"`
 }
+
+// MaxUnschedulableReplicasRequest represents the request that sent by gRPC client to calculate unschedulable replicas.
+type MaxUnschedulableReplicasRequest struct {
```

Comment:
What does the `Max` mean? In my opinion, what the client request is `unschedulable` replicas, there is no notion of a maximum.

Reply:
Right.